rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -54,9 +54,7 @@ task_role_no={{{ idx }}}
 
 printf "%s %s\n%s\n\n" "[INFO]" "ENV" "$(printenv | sort)"
 
-if [[ -n $PAI_CODE_DIR ]]; then
-  hdfs dfs -get $PAI_CODE_DIR || exit 1
-fi
+mv /pai/code/* ./
```

Comment:
>./ [](start = 15, length = 2)

this is a breaking change. can you preserve the previous folder?

Reply:
this isn't a breaking change, user's code is placed under WORK_DIR on the both cases.